src/http/httpClient/HttpClientRequestAdapter.cs

```diff
@@ -54,6 +82,13 @@ public HttpClientRequestAdapter(IAuthenticationProvider authenticationProvider,
             sWriterFactory = serializationWriterFactory ?? SerializationWriterFactoryRegistry.DefaultInstance;
             obsOptions = observabilityOptions ?? new ObservabilityOptions();
             activitySource = ActivitySourceRegistry.DefaultInstance.GetOrCreateActivitySource(obsOptions.TracerInstrumentationName);
+
+#if NET5_0_OR_GREATER
+            this.httpVersion = httpVersion ?? client.DefaultRequestVersion;
+            this.httpVersionPolicy = httpVersionPolicy ?? client.DefaultVersionPolicy;
```

Comment:
if we can get that information from the client in net5+, why do we even need to have a parameter/fields? Can't we simply read it directly from the client that's being passed? and if the consumer wants to tweak things, they can simply create the http client and set it to whatever they need.